docs/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Flexy OSIsoft Changelog
 
+## V3.0

```

Comment:
The format of this changelog *must* be updated to be compatible with the GitHub actions. See https://github.com/hms-networks/flexy-azure-connector/commit/2809ee2e3c204f9154157bb060c0f7c007eac580

Reply:
Would need to be in format:
```suggestion
## Version 3.0.0
```

Reply:
All previous versions need to be updated in the changelog as well. Future release versions also will need to comply with the X.Y.Z version format, as short hand versions as not allowed (i.e 5.1 for 5.1.0)